foreign-key-mapping/README.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: pattern
+title: Foreign-Key-Mapping
+folder: foreign-key-mapping
+permalink: /patterns/foreign-key-mapping/
+categories: Object-Relational Structural
+tags:
+- foreign-key
```

Comment:
The available categories and tags are listed in https://github.com/iluwatar/java-design-patterns/wiki/07.-Categories-and-Tags. If not necessary, please don't make new ones. I suggest we use `Structural` category and `Data access` tag.